### ISSUE TITLE
Improve Pantheon customization to WordPress login form

### ISSUE THREAD

**Issue by ari-gold** [pantheon-systems/WordPress#155]
Pantheon's update to WordPress 4.9.5 included a customization to the WordPress login form, which shows a button to "Return to Pantheon" to help with the first time user experience.

This subsequent update improves upon that customization by only showing the button for platform domains (e.g. live-example.pantheonsite.io) and not for custom or vanity domains.

If you'd like to also remove it for platform domains, add the following to wp-config.php:

```
if ( ! defined( 'RETURN_TO_PANTHEON_BUTTON' ) ) {
	define( 'RETURN_TO_PANTHEON_BUTTON', false );
}
```

**Comment by dylanjameswagner** [comment]
I am not sure this is the best place to mention this, but...

The fact that this was released in tandem with a security release really forced adoption of this feature and required quite a bit of scrambling for projects that demanded attention to be updated for the security release but where this feature was undesired. I had to dig into the code to find a way to manually patch/remove this feature. I don't think this should have been released with a patch level release. 

The long-and-short of it is that there was no announcement of these customizations and no way to disable them at launch. To me, this feature feels rushed/under-tested and under developed at launch.
